server/src/main/java/de/pnp/manager/model/manager/InventoryHandler.java

```diff
@@ -34,7 +37,19 @@ public Container createContainer(String sessionID, String name, Inventory invent
         return container;
     }
 
-    public Collection<Container> getContainers(String sessionID) {
+    public void deleteContainer(String sessionID, String inventoryID) {
+        Collection<Container> containers = this.sessionContainerMap.get(sessionID);
+
+        if (containers != null) {
+            containers.removeIf(c -> c.getInventoryID().equals(inventoryID));
+            manager.getNetworkHandler().activeBroadcast(
+                    new DismissInventoriesMessage(inventoryID, Calendar.getInstance().getTime()),
+                    manager.getNetworkHandler().clientsProperty().filtered(c -> c.hasAccessToInventory(inventoryID))
```

Comment:
this should also remove the inventory from the clients's list of controlled inventories

Reply:
It does, activeBroadcast uses the sendActiveMessage function, which changes the state of a client